src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1610,15 +1611,6 @@ private static Document ParseDocumentPartial(JsonOperationContext context, ref T
             return result;
         }
 
-        public static Document ParseRawDataSectionDocumentWithValidation(JsonOperationContext context, ref TableValueReader tvr, int expectedSize)
```

Comment:
Is there any reason for removing it with the current approach?

Reply:
yes this was calling `ParseDocument()` which is now expects a `DocumentsOperationContext`

Reply:
returned the method